### ISSUE TITLE
Bluesky photo-only replies are interpreted as photo objects

### ISSUE THREAD

**Issue by JoelOtter** [snarfed/granary#621]
So it looks like if a Bluesky reply consists only of a photo, we interpret that as a photo object rather than a comment with an embed - just wanted to check that's what we want? It's broken my webmention handling but perhaps I just need to support this content type too. :)

**Comment by snarfed** [comment]
Hmm! I don't quite follow. Do you mean we miss setting `inReplyTo`? Do you have an example?

**Comment by JoelOtter** [comment]
Yep, so [this post](https://bsky.app/profile/sammgreer.bsky.social/post/3kcoxr7xlg22a) looks like this on Bridgy:

![image](https://github.com/snarfed/granary/assets/2765356/1b4a7123-369e-4a93-940d-3d5a22c756a2)

The resulting webmention looks like this - note the missing `content` field:

```json
{
    "type": "entry",
    "author": {
        "type": "card",
        "name": "Samantha Mackenzie Greer",
        "photo": "https://webmention.io/avatar/cdn.bsky.app/02ef296e2832d094a6908c85353f885fcce81001ae71028cb69006a4ed4dd792.jpg",
        "url": "https://bsky.app/profile/sammgreer.bsky.social"
    },
    "url": "https://bsky.app/profile/sammgreer.bsky.social/post/3kcoxr7xlg22a",
    "published": "2023-10-27T00:04:24+00:00",
    "wm-received": "2023-10-27T00:18:20Z",
    "wm-id": 1731238,
    "wm-source": "https://brid.gy/comment/bluesky/did:plc:ioz4ztghfznx4s5s4jxqiqun/at%253A%252F%252Fdid%253Aplc%253Aioz4ztghfznx4s5s4jxqiqun%252Fapp.bsky.feed.post%252F3kcnzsfvwap2d/at%253A%252F%252Fdid%253Aplc%253Afraj4bml6tl37x3s3syitmzt%252Fapp.bsky.feed.post%252F3kcoxr7xlg22a",
    "wm-target": "https://www.joelotter.com/notes/2023/10/26-signalis/",
    "wm-protocol": "webmention",
    "photo": [
        "https://cdn.bsky.app/img/feed_fullsize/plain/did:plc:fraj4bml6tl37x3s3syitmzt/bafkreiekrafp4kayl3lqgyvnfekvsv7afzetiiibkwczjjkdxrebqvab2y@jpeg"
    ],
    "in-reply-to": "https://www.joelotter.com/notes/2023/10/26-signalis/",
    "wm-property": "in-reply-to",
    "wm-private": false
}
```

I'd expect there to be a `content.html` field which shows how to display that photo - possibly that's not the expected functionality, though? 

Basically I'm asking if this is a bug or a misaligned expectation of mine!

**Comment by snarfed** [comment]
Ah ok!

Strictly speaking, I don't know that this is a bug. But realistically, if we want to be helpful to receivers, you're right, we should probably include content with an HTML `img` tag.

There's a `synthesize_content` kwarg somewhere around `microformats2.activity_to_html` that I would have guessed would do this, and I'm pretty sure we set that to `True`. I wonder why that's not happening.
